.github/workflows/build.yml

```diff
@@ -620,6 +620,70 @@ jobs:
               exit -1
           fi
 
+  xi_test_Linux:
+    runs-on: ubuntu-24.04
+    needs: Linux_Clang18_64bit
+    permissions:
+      contents: read         # Read repository contents
+      checks: write          # Create check runs with test results
+      pull-requests: write   # Post PR comments with test summaries
+      actions: read          # Browse workflow runs and artifacts
+      statuses: write        # Update commit statuses
+    services:
+      mysql:
+        image: mariadb:10.6
+        env:
+          MYSQL_DATABASE: xidb
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=10s --health-retries=10
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          submodules: recursive # For navmeshes
```

Comment:
I think it's a good idea for us to do this here, but not in any of the other CI jobs - I've always been under the impression that this is computationally and bandwidth heavy. If we have workloads where we want to verify that incoming additions to spawn points and pathing are valid, we can now make them inside xi_test 👍